lib/enumerable/map.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_map(block = nil)
+    raise 'Please provide a block' unless block_given?
+
```

Comment:
- [ ] my_map returns an Enumerator if no block is given

Reply:
Added the cases you told me

Reply:
```ruby
- raise 'Please provide a block' unless block_given?
+ return to_enum unless block_given?
```
solved